### ISSUE TITLE
2-site FIM: adjusting to the nearest available map not working

### ISSUE THREAD

**Issue by mhkim01** [USGS-WiM/FIM#141]
MO stuff again. Vally Park, Fenton, and Arnold share a same second site for its 2-site FIM. this is an odd behavior i have not seen before. for some reason, the adjustment (stage combo) to the nearest available map from previous 2-site FIM viewing is repeated when viewing a next 2-site FIM which shows a blank map. of course, if i refresh the mapper, this behavior goes away. 

![image](https://user-images.githubusercontent.com/15948427/39816988-e905b8e2-536a-11e8-81bc-ebf0a285c94c.png)

**Comment by mhkim01** [comment]
this issue might not be something we can address soon. if that's the case, i would like to move on and let MO staff know of the issue, short-term solution (refresh) and get their review process moving forward since they are waiting on me. sound good to you all?

**Comment by njestes** [comment]
@mhkim01 yeah, I think that's the best plan for now. Maybe I can look into it and fix it sometime later, but it will be a while.

**Comment by mhkim01** [comment]
ok, if we can shoot for it being resolved by the time these fims go on the public mapper, that would be great.

**Comment by njestes** [comment]
@mhkim01 maybe. how soon might that be?

**Comment by mhkim01** [comment]
looks like these fims are on the fast track. author thinks about 2 months or less for BAO approval; then a few of weeks for public mapper; it's still an estimate.
